plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Common.hs

```diff
@@ -39,14 +41,12 @@ typecheckAnd
             UPLC.Term Name uni fun () -> a)
     -> CostingPart uni fun -> TPLC.Term TyName Name uni fun () -> m a
 typecheckAnd semvar action costingPart term = TPLC.runQuoteT $ do
-    -- Here we don't use `getDefTypeCheckConfig`, to cover the absurd case where
-    -- builtins can change their type according to their BuiltinSemanticsVariant
+    -- Here we don't use 'getDefTypeCheckConfig', to cover the absurd case where
+    -- builtins can change their type according to their 'BuiltinSemanticsVariant'.
```

Comment:
I do often markdown style backticks in comments. Should I stop doing that?

Reply:
Dunno, Haddock "prefers" `'`, so I use that everywhere in our codebase for consistency together with `@`, but I think Haddock is also fine with backticks as a replacement for `'` (but not `@`), so probably not a big deal. I'd prefer consistency, but it does sound like ensuring consistency would be more pain than just being inconsistent here.

Reply:
`find . -name "*.hs" -exec sed -i 's/`/\'/g'`

What could possibly go wrong?